tests/x11/rmt/rmt_chinese.pm

```diff
@@ -53,6 +53,11 @@ sub set_language_to_Chinese {
     # Logout and login to ensure the configuration for Chinese language take effect.
     handle_relogin;
 
+    # Reserve old folder name and close dialog of requirement for language change to update folder
+    assert_screen 'language-change-required-update-folder';
+    assert_and_click('reserve_old_folder_name');
+    assert_screen([qw(generic-desktop opensuse-welcome)]);
```

Comment:
No need to assert opensuse-welcome here.

Reply:
Thanks, removed the opensuse-wlecome.